src/Settings.cpp

```diff
@@ -269,24 +273,33 @@ void Settings::parseInput(int argc, char** argv)
         case 's':  // Run in P2P server mode
             //-------------------------------------------------------
             mJackTripMode = JackTrip::SERVER;
+            mRunMode      = P2P_SERVER;
```

Comment:
Why do we need mJackTripMode and mRunMode? Isn't that the same?

Reply:
Similar but slightly different. (You'll notice mJackTripMode was never set with -S because it's not needed at this point in the code.) I guess it would be possible to set mJackTripMode to JackTrip::SERVERTOPINGSERVER to show that we're in hub server mode, but the advantage of runModeT is that it matches the more user friendly order that exists in the GUI dropdown. If we slightly repurpose mJackTripMode then we'll need code to translate the order used between the two parts of the program. (Alternatively we could change the order within the jacktripModeT enum if we're sure it won't break any assumptions made about those values elsewhere in the program.)

Reply:
When I get a chance later today, I'll check the code to see if I can safely change the enum order. If I can, I'll consolidate those two variables.

Reply:
Scratch that - I'll just do the translation. It's less efficient, but it means there's less chance that a change in one part of the code will break things elsewhere down the line.

Reply:
(Expect a revision in the not too distant future.)